action.yaml

```diff
@@ -5,16 +5,19 @@ branding:
   icon: 'lock'  
 inputs:
   asoc_key:
-    description: 'API key for connecting to ASoC.'
+    description: 'API key for connecting to the service.'
```

Comment:
This is a minor thing, but for everywhere we say "the service", can you make it "the AppScan service"?